src/components/AuditLogsTable/AuditLogsTable.tsx

```diff
@@ -131,7 +132,7 @@ const AuditLogsTable = () => {
           emptyMsg={emptyMsg}
         />
       )}
-      {auditLogsLoaded && (hasNextPage || page > Number(DEFAULT_PAGE)) ? (
+      {!auditLogsLoading && auditLogsLoaded && (hasNextPage || hasPrevPage) ? (
```

Comment:
I included `!auditLogsLoading` here, because previously, when the audit logs were being fetched, the bottom pagination still appeared. I am not sure if this is the expected behaviour, so let me know if this change should remain.

Reply:
That seems fine to me.